app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> float:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(elfs: list) -> None:
+    [elf.play_elf_song() for elf in elfs]
+
+
+def feast_of_the_dwarves(dwarves: list) -> None:
+    [dwarf.eat_favourite_dish() for dwarf in dwarves]
```

Comment:
List comprehension is good for creating lists, but for loop is good for calling the functions

Reply:
also use more descriptive type annotation for all of this functions,f.e.: players: list[<class_name>]